### DISCUSSION TITLE
Performous 1.3.1 Bugfix-Release

### DISCUSSION THREAD

**Discussion by twollgam** [performous/performous#944] [General]
To make a bugfix release here a list of containing prs and issues:

Issues and prs to do:
- MacOS working again ->  #926 ✅
- UTF-8 Support Windows -> #934 ✅ 
- New theme -> #920 ✅ AND #932 ✅
- Show faulty songs -> #917 ✅
- Song thumbnails disappear -> #949 
- Cache doesn't load correctly -> #946 

Merged prs:
- #935
- #934 
- #932 
- #929 
- #926 
- #925 
- #920 
- #916 

Tested master branch:

- [ ] Singing
- [ ] Theme performous+ Color
- [ ] Theme performous+ Black
- [ ] MacOS

**Comment by Baklap4** [comment]
still gotta write a pr for website repo. And after the faulty songs is merged we can release it :)

i will have some time later tonight to add this :)

**Reply by Baklap4** [reply]
Added PR for website repo: https://github.com/performous/web/pull/21 (please add a comment if i'm missing something there)
